### DISCUSSION TITLE
Any plans for netbird?

### DISCUSSION THREAD

**Discussion by 0xFEEDC0DE64** [SierraSoftworks/tailscale-udm#29] [Ideas]
I am moving away from tailscale to netbird, as they use the real kernel mode wireguard for better performance and easier self-hosting,

any plans to redo all of your tailscale work for netbird? they also offer arm64 binaries on github :)

https://github.com/netbirdio/netbird/releases

**Comment by notheotherben** [comment]
Hi @0xFEEDC0DE64, I'm afraid I currently don't have any plans to do so - primarily because I dogfood this code and test all changes myself before releasing it (and have no personal plans to move away from Tailscale for the time being). That said, I suspect you'll find that most of the logic in this repository is relatively easy to adapt to the Netbird use case and I'd be happy to help you bootstrap a repository doing the same if you're happy to test and maintain it.

On the topic of kernel mode wireguard, I believe Tailscale does offer that on many of their platforms, however I ran into issues with it on the UDM which led to us using userspace networking here.
